documentation/blog/2025-10-03-running-goose-in-containerized-environments/index.md

```diff
@@ -184,78 +183,141 @@ analyze:
 
 I've used this setup on multiple projects, and it's a game-changer for automated code reviews and documentation generation.
 
-## Troubleshooting: Common Gotchas
+## Troubleshooting: Real-World Issues
 
```

Comment:
let's add a quick intro here between the `##` and `###` lines to talk about why troubleshooting can be challenging with Docker, and how you're going to show them some example to get started. Maybe point out that a tip at the bottom of this section will get into advanced debugging.